plugins/commands.js

```diff
@@ -127,6 +127,42 @@ exports.register = (server, options, next) => {
                 }
             }
         }
+    }, {
+        method: 'DELETE',
+        path: '/commands/{namespace}/{name}/{version}',
+        config: {
+            description: 'Delete command binary',
+            notes: 'Delete a script or binary of specific command',
+            tags: ['api', 'commands'],
+            auth: {
+                strategies: ['token'],
+                scope: ['user']
+            },
+            plugins: {
+                'hapi-swagger': {
+                    security: [{ token: [] }]
+                }
+            },
+            handler: (request, reply) => {
+                const { namespace, name, version } = request.params;
+                const id = `${namespace}-${name}-${version}`;
+
+                cache.drop(id, (err) => {
+                    if (err) {
+                        return reply(err);
```

Comment:
I'm unsure what the Screwdriver policy is regarding coverage, but this line isn't covered by tests. The code coverage report generated by `jenkins-mocha` should be located in:

```
./artifacts/coverage/lcov-report/index.html`
```

Personally, I put

```bash
alias cov='open ./artifacts/coverage/lcov-report/index.html'
```

in my `.bashrc` for convenience. Most repositories have a `coveralls` check which ensures the PR doesn't cause a coverage decrease. Example: https://coveralls.io/builds/16510834

Reply:
Looks like we skipped that because the `cache.get` tests the case where the server throws an error when caching is not set up right (which is the only case in which the server throws an error).